File: mo-system/mo-system-service/src/main/java/ink/ckx/mo/system/MyCommandLineRunner.java
package ink.ckx.mo.system;

import ink.ckx.mo.system.service.SysMenuService;
import lombok.RequiredArgsConstructor;
import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/13
 */
@RequiredArgsConstructor
@Component
public class MyCommandLineRunner implements CommandLineRunner {

    private final SysMenuService menuService;

    @Override
    public void run(String... args) {
        menuService.refreshRolePerm();
    }
}

File: mo-admin/mo-admin-service/src/main/kotlin/ink/ckx/mo/admin/MyCommandLineRunner.kt
package ink.ckx.mo.admin

import ink.ckx.mo.admin.service.SysMenuService
import org.springframework.boot.CommandLineRunner
import org.springframework.stereotype.Component

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/13
 */
@Component
class MyCommandLineRunner(
    private val menuService: SysMenuService
) : CommandLineRunner {

    override fun run(vararg args: String) {
        menuService.refreshRolePerm()
    }
}